### ISSUE TITLE
surface area practice worksheet?

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1521]
I think it would be nice to include a worksheet in the surface area lesson where students practice finding surface area.

I encountered this (below) in a desmos activity, which seems to parallel the approach we use in the starter file (but also VERY scaffolded). Kids could fill in tables like this, or we could opt to let them choose their approach. (In the lesson, we give three correct ways to define surface area.)

![image](https://github.com/bootstrapworld/curriculum/assets/87144723/4dfa85a7-884a-43e6-a95f-20a3cf6a9c64)

**Comment by flannery-denny** [comment]
@retabak clarifying that your real ask is just some paper and pencil practice on finding surface area of a rectangular prism, given an image of a rectangular prism. If I write some defend and decide stuff and open-ended stuff that'd work for you?

**Comment by retabak** [comment]
oooh, even better! My suggestion was boring.

Mostly, I think it would be nice to have additional thinking / practice that is not centered around the starter file (which right now fills up the entirety of the lesson).

**Comment by retabak** [comment]
@flannery-denny - the PPSD teachers requested that the launch include a review of the contract for rectangle and how to make differently sized rectangles. They had forgotten how to do this, and were confident that their middle school students would need the review too. 

We could add it and state that if students have made rectangles recently, then teachers can opt to skip the review?

**Comment by retabak** [comment]
Re: the note that begins with "If your students are new to our editor, note that most of the screen is taken up by two large boxes..." 

The lesson prerequisites are simple data types and contracts, so students shouldn't be new to the editor - right? I also noticed that when you refer to the interactions and definitions area throughout the lesson and on worksheets, you helpfully include parentheticals that say "on the left" or "on the right". In other words, the note feels unneeded. I deleted the note, but we can add it back if I'm missing something (which is quite possible!).
